package.json

```diff
@@ -38,6 +38,8 @@
     "@edx/frontend-component-header": "4.2.3",
     "@edx/frontend-platform": "4.5.1",
     "@edx/paragon": "^20.45.0",
+    "@testing-library/jest-dom": "github:testing-library/jest-dom",
```

Comment:
It is enough to just add the requirement in devDependencies. Make sure to run npm install again so that the package-lock.json is ok.